### PR TITLE
refactor(global): INFRA-590 update server utils in miniapps: pass, dev-api, address-service

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**

### CODE REVIEW

File: apps/dev-api/domains/miniapp/schema/PublishB2CAppService.js [hunk 1]
```diff
@@ -320,7 +321,11 @@ const PublishB2CAppService = new GQLCustomSchema('PublishB2CAppService', {
             resolver: async (parent, args, context) => {
                 const { data: { app: { id }, options, environment } } = args
 
-                const app = await B2CApp.getOne(context, { id, deletedAt: null })
+                const app = await B2CApp.getOne(
+                    context,
+                    { id, deletedAt: null },
+                    `id ${EXPORT_FIELDS} name developer createdBy { name } logo { publicUrl originalFilename }`
```
Comment (Alllex202) [inline apps/dev-api/domains/miniapp/schema/PublishB2CAppService.js:327]: mb use `getExportIdField(environment)` or specify these fields explicitly?

```suggestion
const exportIdField = getExportIdField(environment)
// ...
                    `id ${exportIdField} name developer createdBy { name } logo { publicUrl originalFilename }`
```

Comment (Alllex202) [reply apps/dev-api/domains/miniapp/schema/PublishB2CAppService.js:327]: I would indicate it explicitly to make it easier to read

File: apps/dev-api/domains/miniapp/schema/PublishB2CAppService.js [hunk 2]
```diff
@@ -375,7 +380,7 @@ const PublishB2CAppService = new GQLCustomSchema('PublishB2CAppService', {
                         id: options.build.id,
                         deletedAt: null,
                         app: { id: app.id },
-                    })
+                    }, `id ${EXPORT_FIELDS} version data { publicUrl originalFilename mimetype encoding }`)
```
Comment (Alllex202) [inline apps/dev-api/domains/miniapp/schema/PublishB2CAppService.js:383]: ^^

**Comment by Alllex202** [comment]
looks like something is wrong with the tests in dev api

**Review by sitozzz — APPROVED** [review]

**Pull request merged**